cms/CONTENTFUL.md

```diff
@@ -0,0 +1,49 @@
+# Contentful
```

Comment:
Can we create a docs folder?

Reply:
No need with compiled yard docs

Reply:
??? I was just asking if this could go in a docs folder - what does that have to do with yard docs?

Reply:
I mean if you run yard all our markdown ends up in `/doc` already, that's all; admittedly I haven't yet added the `cms` folder to the list, but as with how Rob documented YAML in the `data` dir it means documentation can site beside the relevant bits. There is that `doc` service too which shows live changes as HTML.

Reply:
OK, I just meant it took me more than a beat to find this.  I remembered I'd seen it, but didn't recall that it was in it's own folder.  Also, didn't remember the documentation ind `data`. I'd prefer all the documentation in one place.